keras_nlp/models/gpt_neo_x/gpt_neo_x_backbone_test.py

```diff
@@ -74,17 +73,20 @@ def test_serialization(self):
 
     @pytest.mark.large
     def test_saved_model(self):
-        model_output = self.backbone(self.input_batch)
-        path = os.path.join(self.get_temp_dir(), "model.keras")
-        self.backbone.save(path, save_format="keras_v3")
-        restored_model = keras.models.load_model(path)
+        input_data = tf.constant([" airplane at airport"])
```

Comment:
You should rewrite this to be more like the original form (or copy off other backbone tests).

The issue is that string dtypes to not exist for torch and jax. So `keras.Input(dtype="string", shape=())` will actualy error out on those backends.

Reply:
@mattdangerw  We don't have backbone presets yet.

Reply:
Done !